src/main/java/com/twilio/taskrouter/application/CreateWorkspaceTask.java

```diff
@@ -124,39 +132,53 @@ public static Workflow addWorkflowToWorkspace(WorkspaceFacade workspaceFacade,
     String workflowName = workflowJson.getString("name");
     return workspaceFacade.findWorkflowByName(workflowName)
       .orElseGet(() -> {
-        Map<String, String> properties = new HashMap<>();
-        properties.put("FriendlyName", workflowName);
-        properties.put("AssignmentCallbackUrl", workflowJson.getString("callback"));
-        properties.put("FallbackAssignmentCallbackUrl", workflowJson.getString("callback"));
-        properties.put("TaskReservationTimeout", workflowJson.getString("timeout"));
-        String jsonWorkflowConfig = createWorkFlowJsonConfig(workspaceFacade, workflowJson);
-        properties.put("Configuration", jsonWorkflowConfig);
-        return workspaceFacade.addWorkflow(properties);
+        Map<String, String> newWorkflowParams = new HashMap<>();
+        newWorkflowParams.put("FriendlyName", workflowName);
+        newWorkflowParams.put("AssignmentCallbackUrl", workflowJson.getString("callback"));
+        newWorkflowParams.put("FallbackAssignmentCallbackUrl", workflowJson.getString("callback"));
+        newWorkflowParams.put("TaskReservationTimeout", workflowJson.getString("timeout"));
+        String workflowConfigJson = createWorkFlowJsonConfig(workspaceFacade, workflowJson);
+        newWorkflowParams.put("Configuration", workflowConfigJson);
+        return workspaceFacade.addWorkflow(newWorkflowParams);
```

Comment:
Blank lines are intended to increase the readability of the code. They have to be used like when you're writing plain English.

You can think of having _blocks_ of code. I used to use _assignment_, _processing_, and _return_. This is what I mean:

``` java
// assignment

// processing

// return
```

What do you think?